docs/ios/xcsync.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Xcsync for .NET MAUI
+description: [Xcsync is designed to provide .NET developers with improved support for editing Apple specific files]
+author: haritha-mohan, mcumming
+ms.date: [08/05/2024]
+---
+# Xcsync: improved support for Apple development via .NET
+
+Xcsync is a tool designed to provide developers using the .NET MAUI/.NET Apple SDKs have improved support for editing Apple specific files, such as storyboards and assets. The tool generates a temporary Xcode project from a .NET project, monitors both for changes, and synchronizes the changes between them.
+
+#### How to use Xcsync
+
+The tool has 2 commands right now: generate and sync. Use generate for tracking your changes from your .NET to Xcode and sync for tracking your changes from Xcode to .NET.<br />
```

Comment:
```suggestion
The tool has 2 commands: generate and sync. Use generate for tracking your changes from your .NET to Xcode and sync for tracking your changes from Xcode to .NET.<br />
```

Reply:
So xcsync is installed when I install the iOS, or maui workload? I think making a statement that says "Install .NET 9 Preview 7 or newer" or something is important so the reader knows how to acquire it and/or confirm they have acquired it.

Reply:
the tool is part of the iOS workload specifically..but good point i'll add further clarification here